src/plugins/audiobridge-plugin.js

```diff
@@ -787,6 +798,50 @@ class AudioBridgeHandle extends Handle {
     throw (error);
   }
 
+  /**
+   * 
+   * @param {object} params
+   * @param {number|string} params.room - The involved room
+   * @param {string} [params.secret] - The optional secret needed to manage the room
+   * @returns {} - Returns event as success or error
+   */
+  async muteRoom({ room, secret }) {
+    const body = {
+      request: REQUEST_MUTE_ROOM,
+      room,
+    };
+    if (typeof secret === 'string') body.secret = secret;
+
+    const response = await this.message(body);
+    const { event, data: evtdata } = response._janode || {};
+    if (event === PLUGIN_EVENT.SUCCESS)
+      return evtdata;
+    const error = new Error(`unexpected response to ${body.request} request`);
+    throw (error);
+  }
+
+   /**
+   * 
+   * @param {object} params
+   * @param {number|string} params.room - The involved room
+   * @param {string} [params.secret] - The optional secret needed to manage the room
+   * @returns {} - Returns event as success or error
```

Comment:
Similarly to the mute function, this should return a documented

`Promise<module:audiobridge-plugin~AUDIOBRIDGE_EVENT_UNMUTE_ROOM_RESPONSE>`